jetty-server/src/main/java/org/eclipse/jetty/server/ProxyConnectionFactory.java

```diff
@@ -764,6 +764,11 @@ public ProxyEndPoint(EndPoint endp, InetSocketAddress remote, InetSocketAddress
             _local = local;
         }
 
+        public EndPoint unwrap()
```

Comment:
What do we call the equivalent method in `DecryptedEndPoint`?
We should call it the same... maybe have a `WrappedEndPoint` common interface/class?

Reply:
There is no equivalent method in `DecryptedEndPoint` as `SecureRequestCustomizer` is only interested in the `SslConnection` in that case.